manifests/g/Google/PlatformTools/35.0.2/Google.PlatformTools.yaml

```diff
@@ -0,0 +1,8 @@
+# Created using wingetcreate 1.6.5.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.version.1.9.0.schema.json
+
+PackageIdentifier: Google.PlatformTools
+PackageVersion: 35.0.2
+DefaultLocale: en-US
+ManifestType: version
+ManifestVersion: 1.9.0

```

Comment:
```suggestion
ManifestVersion: 1.7.0
```

Reply:
Manifest 1.9.0 is [live](https://github.com/microsoft/winget-pkgs/pull/195988).